cgmanifest.json

```diff
@@ -525,8 +525,8 @@
         "type": "other",
         "other": {
           "name": "c-ares",
-          "version": "1.17.1",
-          "downloadUrl": "https://c-ares.haxx.se/download/c-ares-1.17.1.tar.gz"
+          "version": "1.18.1",
+          "downloadUrl": "https://c-ares.haxx.se/download/c-ares-1.18.1.tar.gz"
```

Comment:
It looks like this website was recently changed to c-ares.org. Can you update the downloadURL and URLs in the spec?   See #1694